tests/performance-tests/src/performance/init_test.go

```diff
@@ -0,0 +1,67 @@
+package performance_test
+
+import (
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/helpers"
+)
+
+var (
+	DEFAULT_TIMEOUT      = 30 * time.Second
+	CF_PUSH_TIMEOUT      = 2 * time.Minute
+	LONG_CURL_TIMEOUT    = 2 * time.Minute
+	CF_JAVA_TIMEOUT      = 10 * time.Minute
+	DEFAULT_MEMORY_LIMIT = "256M"
+)
+
+const diegoUnsupportedTag = "{NO_DIEGO_SUPPORT} "
+const deaUnsupportedTag = "{NO_DEA_SUPPORT} "
```

Comment:
These two constants are introduced in 40c954dd47210e871a5d222d9d01bee26a6a4455 but I can't see how they're used. Can you explain?